tests/test_crosstalk.py

```diff
@@ -222,18 +238,45 @@ def testTaskAPI(self):
 
         Checks both MeasureCrosstalkTask and the CrosstalkTask.
         """
+        self.setUp_general()
         coeff = np.array(self.crosstalk).transpose()
+        coeffSqr = np.array(self.crosstalk_sqr).transpose()
         config = IsrTask.ConfigClass()
         config.crosstalk.minPixelToMask = self.value - 1
         config.crosstalk.crosstalkMaskPlane = self.crosstalkStr
         isr = IsrTask(config=config)
-        calib = CrosstalkCalib().fromDetector(self.exposure.getDetector(), coeffVector=coeff)
+        calib = CrosstalkCalib().fromDetector(self.exposure.getDetector(),
+                                              coeffVector=coeff,
+                                              coeffSqrVector=coeffSqr)
+        isr.crosstalk.run(self.exposure, crosstalk=calib)
+        self.checkSubtracted(self.exposure)
+
+    def testTaskAPI_NL(self):
```

Comment:
Can you duplicate this test and have it use the `IsrTaskLSST` to get the config and the task?  This should "Just Work," but I'd rather have a test that checks that the two tasks give the same results (the crosstalk task should be identical, so this is just checking that all of the config mechanics are working the same).

Reply:
Sounds good. I had to add `IsrTaskLSST` to the `__init__` in `ip/isr` in a separate commit so I could import it from the test. Tagging @aferte as a heads-up.